core/Commands/TemplateCommand.cs

```diff
@@ -146,6 +147,11 @@ protected async void ExecuteTemplate(string templateName, IReadOnlyCollection<st
                     throw new CliException(
                         $"Template generation failed with status {result.Status}: {result.ErrorMessage}");
                 }
+
+                if(ExecutionContext.Instance.ProjectConfig.RepositoryType == RepositoryType.App)
+                {
+                    Log.Debug($"Make sure you create a business project for your application");
```

Comment:
I'd change this message to something like: Apps need to have an ApplicationVersion project. Make sure you create at least one business project for your application using the addApplicationVersionAssembly

Reply:
Also, verbosity should be at least Information, Debug isn't printed unless you specifically request it with --loglevel